scss/_text.module.scss

```diff
@@ -0,0 +1,55 @@
+h1 {
+  font-family: 'Rubik';
+  font-style: normal;
+  font-weight: 300;
+  font-size: 32px;
+  line-height: 120%;
+}
+
+h2 {
+  font-family: 'PT Mono';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 160%;
+}
+
+h3 {
+  font-family: 'PT Mono';
+  font-style: normal;
```

Comment:
The default value for `font-style` is `normal`. We can remove all of the `font-style` properties.

Reply:
Yeah, i removed all font-style normal properties.